### DISCUSSION TITLE
New version 0.27.0 released!

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/WeConnect-mqtt#72] [Announcements]
## Highlights:
- Much more responsive when controlling charging or climatization

### Changed
- All selective topics are now under "domains" topic ***Warning this is a breaking change in the topic naming, you have to change your subscriptions!***
- Control commands are now much faster feedbacking due to a new feature that tracks if the command was successful
- Updated API to 0.34.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tillsteinbach/WeConnect-mqtt/releases/tag/v0.27.0'>0.27.0</a>.</em>

**Comment by diplix** [comment]
thanks for your relentless refactoring and updates to this helper, i really like the thorowness of your implementaion. and it works very well! 
as i understand the climatisation has some kind of a callback to indicate the climatisation status without needing an explicit poll. that worked in pre 0.27.x versions already. and now it’s supposed to work better and faster (haven’t tested it yet). are there any other  data point that are „callbackable“?
[this](https://github.com/tillsteinbach/VWsFriend/blob/9498ced60827d8e2c22f43706a406ea5e62b59d5/vwsfriend/vwsfriend/homekit/charging.py#L30) looks as if it would be possible for charging state changes? or does it presently only work for climatisation and starting/ending charging sessions?

BTW, i use WeConnect-mqtt with home-assistant, would this be a good place to drop my configuration for others interested in using this with home-assistant? configuring this for home-assistant-mqtt is a little tedious. i based my configuration on [this repo](https://github.com/DarkFox/ha-config/blob/master/packages/integrations/vw.yaml) but needed to make some changes to make it work with WeConnect-mqtt.

**Reply by tillsteinbach** [reply]
Unfortunately it is still polling and not a real callback. But the requests response allows to determine that the car is working on the request and during that time I increase the polling frequency for that particular domain.
It works for controlling the climatisation and starting/stopping charging. Unfortunately there is no way right now to get a notification when any other state changes.

Regarding your home-assistant configuration, this is great, if you like you can open a new page in the wiki similar to this one: https://github.com/tillsteinbach/WeConnect-mqtt/wiki/FHEM-examples that would be a great starting point for other users.
